geoips/interfaces/base.py

```diff
@@ -670,16 +714,16 @@ def get_plugin(self, name):
         # Find the plugin module
```

Comment:
https://github.com/NRLMMD-GEOIPS/geoips/issues/436
https://github.com/NRLMMD-GEOIPS/geoips/pull/419#issuecomment-1924473740

It appears that get_plugin is the same for BaseYamlInterface and BaseModuleInterface. We should check whether this is true and, if so, combine them on BaseInterface.  

get_plugin is defined right here on base.py, so not sure why they are subclassed on YAML and Module if they are the same..

Reply:
Since I opened #436 to investigate this, I think we should resolve this conversation.

Reply:
Ah, oops, I opened an issue for this too. I'll close it as duplicate and close this.